solid-terms.ttl

```diff
@@ -1,212 +1,283 @@
-@base <http://www.w3.org/ns/solid/terms> .
-@prefix : <#> .
-@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
-@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
-@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
-@prefix owl: <http://www.w3.org/2002/07/owl#> .
-@prefix cc: <https://creativecommons.org/ns#> .
-@prefix dcterms: <http://purl.org/dc/terms/> .
-@prefix log: <http://www.w3.org/2000/10/swap/log#> .
-@prefix ldp: <http://www.w3.org/ns/ldp#> .
-
-<>
+# See details within this document for linkage to specification and purpose.
+# This ontology file is a non-normative supporting document.
+
+# Solid terms ontology
+
+## Ontology preferred namespace prefix and URI
+prefix solid: <http://www.w3.org/ns/solid/terms#>
+
+## Other namespaces used
+### Core W3C ontologies (OWL 2 reserved vocabulary)
+prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+prefix xsd: <http://www.w3.org/2001/XMLSchema#>
+### Other ontologies
+prefix dc: <http://purl.org/dc/terms/>
+prefix ldp: <http://www.w3.org/ns/ldp#>
+prefix log: <http://www.w3.org/2000/10/swap/log#>
+prefix vann: <http://purl.org/vocab/vann/>
+
+
+## Ontology
+<http://www.w3.org/ns/solid/terms>
     a owl:Ontology ;
-    cc:attributionURL <> ;
-    cc:license <https://creativecommons.org/publicdomain/zero/1.0/> ;
-    dcterms:issued "2015-11-16"^^xsd:date ;
-    dcterms:modified "2018-01-24"^^xsd:date ;
-    rdfs:label "Solid terms"@en .
+    rdfs:label "Solid Terms ontology"@en ;
```

Comment:
```suggestion
    rdfs:label "Solid Terms"@en ;
```

Prefer to keep it simple. "Ontology" scares people =)

Reply:
That makes me so sad. I love this word. But, I think in the greater scheme of things, we do not label classes "Account Class", so frankly, I think I let my love take over my best judgement on this one.
I'll try to sneak the word Ontology in other places ^^,